### PR TITLE
Add pk script

### PR DESCRIPTION
**Pull request by Ic3Sandy**

### CODE REVIEW

File: packages/bitcore-node/src/models/baseTransaction.ts [hunk 1]
```diff
@@ -36,6 +36,7 @@ export interface ITransaction {
   weight?: number;
   vsize?: number;
   receipt?: Array<ITransactionReceipt>;
+  vinScriptSig?: Array<string>;
```
Comment (thebevrishot) [inline packages/bitcore-node/src/models/baseTransaction.ts]: Should we put this field to coin model instead?

**Review by thebevrishot — CHANGES_REQUESTED** [review]

File: packages/bitcore-node/src/models/baseTransaction.ts (hunk 1 above)
Comment (Ic3Sandy) [reply packages/bitcore-node/src/models/baseTransaction.ts]: ok kub

File: packages/bitcore-node/src/models/coin.ts [hunk 2]
```diff
@@ -21,6 +22,7 @@ export interface ICoin {
   spentHeight: number;
   confirmations?: number;
   sequenceNumber?: number;
+  vinScriptSig?: string;
```
Comment (thebevrishot) [inline packages/bitcore-node/src/models/coin.ts]: Should this be buffer?

File: packages/bitcore-node/src/routes/api/tx.ts [hunk 3]
```diff
@@ -129,7 +129,7 @@ router.get('/:txid/coins', (req, res, next) => {
     chain = chain.toUpperCase();
     network = network.toLowerCase();
     ChainStateProvider.getCoinsForTx({ chain, network, txid })
-      .then((coins) => {
+      .then(async (coins) => {
```
Comment (thebevrishot) [inline packages/bitcore-node/src/routes/api/tx.ts]: Do this have to be async?

File: packages/bitcore-node/src/models/coin.ts (hunk 2 above)
Comment (Ic3Sandy) [reply packages/bitcore-node/src/models/coin.ts]: ok kub

File: packages/bitcore-node/src/routes/api/tx.ts (hunk 3 above)
Comment (Ic3Sandy) [reply packages/bitcore-node/src/routes/api/tx.ts]: I will remove this kub.

**Review by thebevrishot — APPROVED** [review]
LGTM

**Pull request merged**